lib/convection/control/stack.rb

```diff
@@ -105,7 +105,7 @@ def []=(key, value)
       end
 
       def get(*args)
-        @attributes.get(*args)
+        @attributes.get(*args, "** Warning: Could not resolve value [#{args[1]}] for stack [#{args[0]}] **")
```

Comment:
[This](https://github.com/rapid7/convection/blob/master/lib/convection/model/attributes.rb#L43-L45) is really where the issue is, though, the question is do we want it to fail during template render or during apply/diff stage.  @jmanero-r7 Care to weigh in here?